packages/cli/src/environments/source-control/source-control-helper.ee.ts

```diff
@@ -1,10 +1,13 @@
 import { generateKeyPairSync } from 'crypto';
 import { constants as fsConstants, mkdirSync, accessSync } from 'fs';
+import { ApplicationError } from 'n8n-workflow';
+import { ok } from 'node:assert/strict';
```

Comment:
Not something for this PR, but do we maybe want to investigate a more flexible solution for asserts? Or at least set some guidelines (e.g. whether to use `assert/strict` or not).

Reply:
Good thinking. I think for 99% of the cases we are checking
1. if something is defined and not `null` or `undefined` or
2. if some condition is true.

I think for that `node:assert` is fulfils the need. But yeah would be good to align.